test_util/test_aws_cf.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+"""Deploys DCOS AWS CF template and then runs integration_test.py
+
+The following environment variables control test procedure:
+
+DCOS_TEMPLATE_URL: string
+    The template to be used for deployment testing
+
+CI_FLAGS: string (default=None)
+    If provided, this string will be passed directly to py.test as in:
+    py.test -vv CI_FLAGS integration_test.py
+
+TEST_ADD_ENV_*: string (default=None)
+    Any number of environment variables can be passed to integration_test.py if
+    prefixed with 'TEST_ADD_ENV_'. The prefix will be removed before passing
+"""
+import logging
+import os
+import random
+import stat
+import string
+import sys
+from contextlib import closing
+
+import test_util.aws
+import test_util.test_runner
+from gen.calc import calculate_environment_variable
+from ssh.ssh_tunnel import SSHTunnel
+
+LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
+logging.basicConfig(format=LOGGING_FORMAT, level=logging.DEBUG)
+log = logging.getLogger(__name__)
+
+
+def check_environment():
+    """Test uses environment variables to play nicely with TeamCity config templates
+    Grab all the environment variables here to avoid setting params all over
+
+    Returns:
+        object: generic object used for cleanly passing options through the test
+
+    Raises:
+        AssertionError: if any environment variables or resources are missing
+            or do not conform
+    """
+    options = type('Options', (object,), {})()
+
+    # Defaults
+    options.variant = os.getenv('DCOS_VARIANT', 'default')
```

Comment:
There is no such variant `default` in DC/OS. There is the python `None` variant, which wen stringified is the empty string, and we have a downstream which is an `ee` variant